### PR TITLE
Tune model params

### PR DESCRIPTION
**Pull request by leonardcser**
Test first locally with a small dataset, then use GPUs on cluster to train models.

### CODE REVIEW

**Comment by leonardcser** [comment]
# Params workflow vs. main

| Path      | Param                            | main   | workspace   |
|-----------|----------------------------------|--------|-------------|
| params.py | MULTIPROCESSING                  | True   | False       |
| params.py | PrepareLocalizeParams.MAX_IMAGES | 100000 | 256         |
| params.py | PrepareOCRParams.MAX_IMAGES      | 250000 | 256         |
| params.py | PrepareStackParams.MAX_IMAGES    | 5000   | 256         |
| params.py | TrainLocalizeParams.EPOCHS       | 3      | 1           |
| params.py | TrainOCRParams.EPOCHS            | 2      | 1           |

# Metrics workflow vs. main

| Path                               | Metric          | main     | workspace   | Change   |
|------------------------------------|-----------------|----------|-------------|----------|
| out/evaluations/stack/metrics.json | val_acc         | 0.84355  | 0.0         | -0.84355 |
| out/runs/localize/metrics.json     | step            | 2        | 0           | -2       |
| out/runs/localize/metrics.json     | train.loss      | 0.00131  | 0.0044      | 0.0031   |
| out/runs/localize/metrics.json     | validation.loss | 0.00019  | 0.01635     | 0.01616  |
| out/runs/ocr/metrics.json          | step            | 1        | 0           | -1       |
| out/runs/ocr/metrics.json          | train.loss      | -0.53072 | 0.82487     | 1.35559  |
| out/runs/ocr/metrics.json          | validation.loss | -0.34747 | 3.15187     | 3.49934  |

# Models

## Localize Model Validation Loss



## OCR Model Validation Loss

**Comment by leonardcser** [comment]
# Params workflow vs. main

| Path      | Param                            | main   | workspace   |
|-----------|----------------------------------|--------|-------------|
| params.py | MULTIPROCESSING                  | True   | False       |
| params.py | PrepareLocalizeParams.MAX_IMAGES | 100000 | 256         |
| params.py | PrepareOCRParams.MAX_IMAGES      | 250000 | 256         |
| params.py | PrepareStackParams.MAX_IMAGES    | 5000   | 256         |
| params.py | TrainLocalizeParams.EPOCHS       | 3      | 1           |
| params.py | TrainOCRParams.EPOCHS            | 2      | 1           |

# Metrics workflow vs. main

| Path                               | Metric          | main     | workspace   | Change   |
|------------------------------------|-----------------|----------|-------------|----------|
| out/runs/ocr/metrics.json          | step            | 1        | 0           | -1       |
| out/runs/ocr/metrics.json          | train.loss      | -0.53072 | 0.82487     | 1.35559  |
| out/runs/ocr/metrics.json          | validation.loss | -0.34747 | 3.15187     | 3.49934  |
| out/evaluations/stack/metrics.json | val_acc         | 0.84355  | 0.0         | -0.84355 |
| out/runs/localize/metrics.json     | step            | 2        | 0           | -2       |
| out/runs/localize/metrics.json     | train.loss      | 0.00131  | 0.0044      | 0.0031   |
| out/runs/localize/metrics.json     | validation.loss | 0.00019  | 0.01635     | 0.01616  |

# Models

## Localize Model Validation Loss



## OCR Model Validation Loss

**Comment by leonardcser** [comment]
# Params workflow vs. main

| Path      | Param                            | main   | workspace   |
|-----------|----------------------------------|--------|-------------|
| params.py | MULTIPROCESSING                  | True   | False       |
| params.py | PrepareLocalizeParams.MAX_IMAGES | 100000 | 256         |
| params.py | PrepareOCRParams.MAX_IMAGES      | 250000 | 256         |
| params.py | PrepareStackParams.MAX_IMAGES    | 5000   | 256         |
| params.py | TrainLocalizeParams.EPOCHS       | 3      | 1           |
| params.py | TrainOCRParams.EPOCHS            | 2      | 1           |

# Metrics workflow vs. main

| Path                               | Metric          | main     | workspace   | Change   |
|------------------------------------|-----------------|----------|-------------|----------|
| out/runs/localize/metrics.json     | step            | 2        | 0           | -2       |
| out/runs/localize/metrics.json     | train.loss      | 0.00131  | 0.0044      | 0.0031   |
| out/runs/localize/metrics.json     | validation.loss | 0.00019  | 0.01635     | 0.01616  |
| out/evaluations/stack/metrics.json | val_acc         | 0.84355  | 0.0         | -0.84355 |
| out/runs/ocr/metrics.json          | step            | 1        | 0           | -1       |
| out/runs/ocr/metrics.json          | train.loss      | -0.53072 | 0.82487     | 1.35559  |
| out/runs/ocr/metrics.json          | validation.loss | -0.34747 | 3.15187     | 3.49934  |

# Models

## Localize Validation Loss



## OCR Validation Loss

**Comment by leonardcser** [comment]
# Params workflow vs. main

| Path   | Param   | main   | workspace   |
|--------|---------|--------|-------------|

# Metrics workflow vs. main

| Path                               | Metric          | main     | workspace   | Change   |
|------------------------------------|-----------------|----------|-------------|----------|
| out/runs/localize/metrics.json     | train.loss      | 0.00131  | 0.00143     | 0.00012  |
| out/runs/localize/metrics.json     | validation.loss | 0.00019  | 0.00015     | -3e-05   |
| out/evaluations/stack/metrics.json | val_acc         | 0.84355  | 0.83594     | -0.00761 |
| out/runs/ocr/metrics.json          | train.loss      | -0.53072 | -0.54302    | -0.0123  |
| out/runs/ocr/metrics.json          | validation.loss | -0.34747 | -0.34904    | -0.00157 |

# Models

## Localize Validation Loss



## OCR Validation Loss
